tests/pr_test.go

```diff
@@ -54,6 +55,38 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "advanced-key-protect",
+		TarIncludePatterns: []string{
+			"*.tf",
+			"modules/*/*.tf",
```

Comment:
there is no modules folder in this repo so I think thats why test failed in pipeline. You will want to add `advancedExampleTerraformDir` here

Reply:
@removed this folder. please re-run